### PR TITLE
added .idea to gitignore and changed readme to right project

### PR DESCRIPTION
**Pull request by TimV22**

### CODE REVIEW

File: README.md
Comment (l-huisman) [inline README.md:1]: 👍🏻

File: .gitignore
Comment (l-huisman) [inline .gitignore:1]: 🥇

**Review by l-huisman — APPROVED** [review]

File: README.md
Comment (TimV22) [reply README.md:1]: 👎

File: .gitignore
Comment (TimV22) [reply .gitignore:1]: 🥈

**Pull request merged**